x-pack/plugins/observability/public/pages/slo_edit/components/slo_edit_form.tsx

```diff
@@ -57,6 +58,8 @@ export function SloEditForm({ slo }: Props) {
   const history = useHistory();
   const { search } = useLocation();
 
+  const { data: rules, isLoading } = useFetchRulesForSlo({ sloIds: [slo?.id || ''] });
+
```

Comment:
useFetchRulesForSlo uses `enabled: Boolean(sloIds)`, if we provide `[""]` it will return true, we should change the hook logic to maybe take a `sloIds: SloId[] | undefined` and in this case change this usage:

```suggestion
  const { data: rules, isLoading } = useFetchRulesForSlo({ sloIds: slo?.id ? [slo.id] : undefined });
```